README.md

```diff
@@ -52,6 +52,35 @@ This project comes with a command that generate the JavaScript version of all yo
 
     php artisan lang:js -c
 
+**(optional)Use [gulp](http://gulpjs.com/) to publish:**
```

Comment:
Can you put the ‘(optional)’ part in the end of the heading. I think it will look better. :grinning: